pkg/deployer/container/container_reconcile.go

```diff
@@ -618,12 +621,13 @@ func (c *Container) SyncConfiguration(ctx context.Context, defaultLabels map[str
 
 // SyncExport syncs the export secret from the wait container to the deploy item export.
 func (c *Container) SyncExport(ctx context.Context) error {
-	c.log.Debug("Sync export to landscaper cluster", "deployitem", kutil.ObjectKey(c.DeployItem.Name, c.DeployItem.Namespace).String())
+	log, ctx := logging.FromContextOrNew(ctx, nil)
+	log.Debug("Sync export to landscaper cluster", "deployitem", kutil.ObjectKey(c.DeployItem.Name, c.DeployItem.Namespace).String())
```

Comment:
Should the key here be `KeyResource`?
If not, i think we should create a constant for "deployItem".